applications/DEMApplication/custom_elements/spheric_particle.cpp

```diff
@@ -914,6 +915,9 @@ void SphericParticle::ComputeBallToBallContactForce(SphericParticle::ParticleDat
                                   GlobalElasticContactForce, GlobalElasticExtraContactForce, TotalGlobalElasticContactForce, ViscoDampingLocalContactForce, cohesive_force, other_ball_to_ball_forces, r_elastic_force, r_contact_force, i, r_process_info);
             //TODO: make different AddUpForces for continuum and discontinuum (different arguments, different operations!)
 
+            // Store contact information needed for other processes
+            StoreBallToBallContactInfo(r_process_info, data_buffer, data_buffer.mpOtherParticle, GlobalContactForce, sliding);
```

Comment:
Shouldn't this method be after the rotation forces have been computed? Otherwise we can only store information which does not contain "rotation forces"

Reply:
Yes, I was interested only on the normal forces, now I moved it to the end to be more generic.